src/vs/platform/contextview/browser/contextMenuHandler.css

```diff
@@ -4,6 +4,6 @@
  *--------------------------------------------------------------------------------------------*/
 
 .context-view .monaco-menu {
-	min-width: 130px;
+	min-width: 160px;
 }
```

Comment:
I think we can remove this file since you've added the rule to the menu widget css and this one gets overruled

Reply:
Removed